index.html

```diff
@@ -1 +1,25 @@
-KJ Loving
\ No newline at end of file
+KJ Loving
+<!DOCTYPE html>
+    <html>
+        <head>
+            <title>KJ's Portfolio</title>
+            <meta name="beginner code portfolio" content="Kj's beginner code projects">
+        </head>
+        <body>
+            <section id = "About">
```

Comment:
You don't need spaces around the `=` for HTML attributes:
```suggestion
            <section id="About">
```